### ISSUE TITLE
Feature request - Microdata

### ISSUE THREAD

**Issue by ghost** [GCX/acf-address-field#6]
Hi!

To start with - thanks for a great plugin!

I would very much like to see a feature where you can incorporate microdata in the output of the adress.

Cheers,
Markus

**Comment by nturbanov** [comment]
Agreed, this would be a big advantage. However, instead of Microdata, I believe Microformats has wider support and since the HTML5 spec is apparently compatible with it, seems the better choice.

http://microformats.org/wiki/hcard
